### ISSUE TITLE
TabStrip Lacks a Focus Indicator

### ISSUE THREAD

**Issue by tjvantoll** [telerik/kendo-ui-core#139]
When you focus a TabStrip you have no visual indication that the control has focus. You can recreate this with a simple TabStrip, e.g. http://trykendoui.telerik.com/UPag.
